gulpfile.js

```diff
@@ -12,17 +12,30 @@ const readFileAsync = promisify(fs.readFile);
 const writeFileAsync = promisify(fs.writeFile);
 
 // Scripts bundled into a single vendor.js
+// You can choose to remove scripts in case they are not needed
+// for your components.
 const scriptsDir = path.join(__dirname, './scripts')
 const scriptFiles = [
+  // Adobe AEM library used by fullscreen.support.js
   path.join(scriptsDir, 'dpsHTMLGestureAPI.min.js'),
+
+  // JQuery libraries used by below support scripts
   path.join(scriptsDir, 'jquery.js'),
```

Comment:
Just wonder about jquery version, we use 2.x here, but in the editor we use 3.x, right? But it seems to work ok :)

Reply:
Looks like JQuery Mobile does not support version 3 yet, which is currently used for the tap event in the fullscreen image script.

The different versions itself should not really matter that much, as the editor should not rely on the jquery version used in the article.

Reply:
I've updated JQuery to version 2.2.4, so it's at least the latest 2.x version.